cmd/powd/main.go

```diff
@@ -280,6 +282,7 @@ func setupFlags() error {
 	pflag.String("ipfsapiaddr", "/ip4/127.0.0.1/tcp/5001", "IPFS API endpoint multiaddress. (Optional, only needed if FFS is used)")
 	pflag.Int64("walletinitialfund", 4000000000000000, "FFS initial funding transaction amount in attoFIL received by --lotusmasteraddr. (if set)")
 	pflag.String("gatewayhostaddr", "0.0.0.0:7000", "Gateway host listening address")
+	pflag.String("ipfsrevproxyaddr", "127.0.0.1:6003", "IPFS reverse proxy listen address")
```

Comment:
I'm not a big fan of having this feature enabled, so my first thought was doing `""` to let be disabled.
Since some people have requested this featured, I opted to leave it open for localhost only.